session/session.go

```diff
@@ -252,9 +253,9 @@ func (s *session) OpenCh(pctx context.Context, openingBalInfo perun.BalInfo, app
 	}
 
 	sanitizeBalInfo(openingBalInfo)
-	parts, err := retrieveParts(openingBalInfo.Parts, s.contacts)
+	parts, err := retrieveParts(openingBalInfo.Parts, s.idProvider)
 	if err != nil {
-		s.Error(err, "retrieving channel participants using session contacts")
+		s.Error(err, "retrieving channel participants using session idProvider")
```

Comment:
```suggestion
		s.Error(err, "retrieving channel participant IDs using session ID Provider")
```
Using ID Provider as two words in comments and error messages.